app/views/trainees/lead_schools/edit.html.erb

```diff
@@ -0,0 +1,15 @@
+<h1 class="govuk-heading-l">Lead school</h1>
```

Comment:
![image](https://user-images.githubusercontent.com/470137/116436244-1e07c400-a844-11eb-8dad-614f7272ba5e.png)

Reply:
added title and heading